addons/web/static/src/views/fields/percent_pie/percent_pie_field.scss

```diff
@@ -9,6 +9,15 @@
     }
 }
 
+.o_list_renderer .o_list_table .o_list_number{
+    .o_field_percent_pie{
+        display: inline-flex !important;
```

Comment:
While this aligns the values on the right, this makes the pies unaligned and I think it does look better without this property, that's just my personal taste tho, you might want to validate that.